app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -221,6 +249,30 @@ public void onMapReady(GoogleMap googleMap) {
     }
 
     /**
+
```

Comment:
I didn't write this code. And yes that's how it would work however for this app there are only 2. The entire application functions this way right now